packages/core/cms/faststore/sections.json

```diff
@@ -1033,7 +1033,10 @@
         },
         "privacyPolicy": {
           "title": "Privacy Policy Disclaimer",
-          "type": "string"
+          "type": "string",
+          "widget": {
+            "ui:widget": "draftjs-rich-text"
```

Comment:
woww, nice!
out of curiosity: where we can find the available widgets?

<img width="621" alt="Screenshot 2023-05-31 at 19 06 00" src="https://github.com/vtex/faststore/assets/11325562/9c91f873-acb3-4bf2-af36-283d93ac9612">

Reply:
I just got an old example from Carrefour hehe. But I think those are the available widgets: https://github.com/vtex/admin-cms/blob/01e3eee65e4d9251d8367218213de90e56f712d2/react/typings/json-schema.d.ts#L14

Reply:
https://www.faststore.dev/how-to-guides/cms/vtex-headless-cms/Widgets

Reply:
Nicee! ty.